contracts/bonding/BondingManager.sol

```diff
@@ -101,10 +101,10 @@ contract BondingManager is ManagerProxyTarget, IBondingManager {
     uint256 public nextRoundTotalActiveStake;
 
     // Check if sender is TicketBroker
-    modifier onlyTicketBroker() {
+    modifier onlyJobsManager() {
```

Comment:
Perhaps we should consider registering `TicketBroker` as both `TicketBroker` and `JobsManager`

This would reduce the amount of possible confusion. Additionally since `updateTranscoderWithFees` where this modifier is used does not involve any direct `Minter` logic, so I would keep the scope of reverting modifier name changes to the `Minter` contract. WDYT?